### PR TITLE
New version: LinwoodCloud.Butterfly.Nightly version 1.6.0-alpha.1

### PR DESCRIPTION
**Pull request by CodeDoctorDE**
###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/89722)

### CODE REVIEW

**Review by russellbanks — DISMISSED** [review]

**Comment by stephengillie** [comment]
Hi @CodeDoctorDE,

This prompt pops up during install, despite `scope: machine` being set in `LinwoodCloud.Butterfly.Nightly.installer.yaml`. I couldn't find an installer switch to evade this prompt. Are you aware of one? 

image

**Comment by CodeDoctorDE** [comment]
Oh I see, I use inno setup. I will take a look how I can bypass it

**Comment by CodeDoctorDE** [comment]
I found out we need to add `/CURRENTUSER` to the command args: https://jrsoftware.org/ishelp/index.php?topic=setup_privilegesrequiredoverridesallowed
How can I add it?

**Comment by stephengillie** [comment]
@CodeDoctorDE,

Today, Defender finds a trojan in the installer. 
image

**Comment by CodeDoctorDE** [comment]
Hmm, that can't be the case. The app is 100% open-source and I build the app and the installer with github actions

File: manifests/l/LinwoodCloud/Butterfly/Nightly/1.6.0-alpha.1/LinwoodCloud.Butterfly.Nightly.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+MinimumOSVersion: 10.0.0.0
+InstallerType: inno
+Scope: machine
+ReleaseDate: 2022-11-25
+Installers:
+- Architecture: neutral
+  InstallerUrl: https://github.com/LinwoodCloud/Butterfly/releases/download/v1.6.0-alpha.1/linwood-butterfly-windows-setup.exe
+  InstallerSha256: 95AB80E1C47ED0FEDB218858AD650F85D6114EFB5F5E65004E22610E82328D9E
+  InstallerSwitches:
+    Silent: /CURRENTUSER
```
Comment (russellbanks) [inline manifests/l/LinwoodCloud/Butterfly/Nightly/1.6.0-alpha.1/LinwoodCloud.Butterfly.Nightly.installer.yaml:15]:
```suggestion
    Custom: /CURRENTUSER
```

**Review by russellbanks — CHANGES_REQUESTED** [review]

**Review by russellbanks — CHANGES_REQUESTED** [review]
The scope needs setting for each installer. It is currently set to machine whilst the installer switch is shown as currentuser. It would be best to keep the machine scope, adding a second entry with the same url with a switch for machine installation.

File: manifests/l/LinwoodCloud/Butterfly/Nightly/1.6.0-alpha.1/LinwoodCloud.Butterfly.Nightly.installer.yaml [hunk 2]
```diff
@@ -0,0 +1,17 @@
+# Created with YamlCreate.ps1 v2.2.1 using InputObject 🤖 $debug=QUSU.CRLF.7-2-7
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.2.0.schema.json
+
+PackageIdentifier: LinwoodCloud.Butterfly.Nightly
+PackageVersion: 1.6.0-alpha.1
+MinimumOSVersion: 10.0.0.0
+InstallerType: inno
+Scope: machine
+ReleaseDate: 2022-11-25
+Installers:
```
Comment (russellbanks) [inline manifests/l/LinwoodCloud/Butterfly/Nightly/1.6.0-alpha.1/LinwoodCloud.Butterfly.Nightly.installer.yaml:10]:
```suggestion
Installers:
- Architecture: neutral
  Scope: machine
  InstallerUrl: https://github.com/LinwoodCloud/Butterfly/releases/download/v1.6.0-alpha.1/linwood-butterfly-windows-setup.exe
  InstallerSha256: 95AB80E1C47ED0FEDB218858AD650F85D6114EFB5F5E65004E22610E82328D9E
  InstallerSwitches:
    Custom: /ALLUSERS
```

File: manifests/l/LinwoodCloud/Butterfly/Nightly/1.6.0-alpha.1/LinwoodCloud.Butterfly.Nightly.installer.yaml [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+- Architecture: neutral
```
Comment (russellbanks) [inline manifests/l/LinwoodCloud/Butterfly/Nightly/1.6.0-alpha.1/LinwoodCloud.Butterfly.Nightly.installer.yaml:11]:
```suggestion
- Architecture: neutral
  Scope: user
```

File: manifests/l/LinwoodCloud/Butterfly/Nightly/1.6.0-alpha.1/LinwoodCloud.Butterfly.Nightly.installer.yaml [hunk 4]
```diff
@@ -0,0 +1,17 @@
+# Created with YamlCreate.ps1 v2.2.1 using InputObject 🤖 $debug=QUSU.CRLF.7-2-7
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.2.0.schema.json
+
+PackageIdentifier: LinwoodCloud.Butterfly.Nightly
+PackageVersion: 1.6.0-alpha.1
+MinimumOSVersion: 10.0.0.0
+InstallerType: inno
+Scope: machine
```
Comment (russellbanks) [inline manifests/l/LinwoodCloud/Butterfly/Nightly/1.6.0-alpha.1/LinwoodCloud.Butterfly.Nightly.installer.yaml:8]:
```suggestion
```

**Review by russellbanks — CHANGES_REQUESTED** [review]

**Comment by CodeDoctorDE** [comment]
Oh thanks. I think two installers will confuse people.

**Comment by CodeDoctorDE** [comment]
Finally 🎉

**Review by russellbanks — APPROVED** [review]

**Review by ghost — APPROVED** [review]

**Pull request merged**